op-chain-ops/deployer/init.go

```diff
@@ -105,7 +105,12 @@ func Init(cfg InitConfig) error {
 	for _, l2ChainID := range cfg.L2ChainIDs {
 		l2ChainIDBig := l2ChainID.Big()
 		intent.Chains = append(intent.Chains, &state.ChainIntent{
-			ID: l2ChainID,
+			ID:                         l2ChainID,
+			BaseFeeVaultRecipient:      common.Address{},
+			L1FeeVaultRecipient:        common.Address{},
+			SequencerFeeVaultRecipient: common.Address{},
```

Comment:
These will automatically default to their zero value if not specified. So nothing wrong with explicitly setting the values here, but the effect is the same as if they are not specified

Reply:
Okay, having them as zero for now is okay. I'll keep the `common.Address{}` in unless there's a reason I shouldn't.